Mathlib/FieldTheory/Finite/GaloisField.lean

```diff
@@ -222,7 +223,47 @@ end GaloisField
 
 namespace FiniteField
 
-variable {K : Type*} [Field K] [Fintype K] {K' : Type*} [Field K'] [Fintype K']
+variable {K K' : Type*} [Field K] [Field K']
+
+theorem algebraMap_norm_eq_pow [Algebra K K'] [Fintype K] [Fintype K'] {x : K'} :
+    algebraMap K K' (Algebra.norm K x) = x ^ ((Fintype.card K' - 1) / (Fintype.card K - 1)) := by
+  simp_rw [Algebra.norm_eq_prod_automorphisms,
+    ← (bijective_frobeniusAlgEquivOfAlgebraic_pow K K').prod_comp, AlgEquiv.coe_pow,
+    coe_frobeniusAlgEquivOfAlgebraic, pow_iterate, Finset.prod_pow_eq_pow_sum,
+    Fin.sum_univ_eq_sum_range, Nat.geomSum_eq Fintype.one_lt_card, ← Module.card_eq_pow_finrank]
+
+variable (K K')
+
+theorem unitsMap_norm_surjective [Algebra K K'] [Finite K'] :
+    Function.Surjective (Units.map <| Algebra.norm K (S := K')) :=
```

Comment:
I've extracted
```
lemma surjective_of_card_ker_le_div {G M : Type*} [Group G] [Group M] [Finite G] [Finite M]
    (f : G →* M) (h : Nat.card f.ker ≤ Nat.card G / Nat.card M) : Function.Surjective f
```
(which doesn't require cyclicity)
so what remains in this proof is just bounding the order of the kernel which indeed has `IsCyclic.card_pow_eq_one_le` as the central piece.